src/Models/T1_relaxometry/vfa_t1.m

```diff
@@ -93,19 +93,16 @@
             TR = obj.Prot.VFAData.Mat(1,2);
             E = exp(-TR/x.T1);
             Smodel = x.M0*sin(flipAngles/180*pi)*(1-E)./(1-E*cos(flipAngles/180*pi));
-
         end
 
        function FitResult = fit(obj,data)
             % T1 and M0
             flipAngles = (obj.Prot.VFAData.Mat(:,1))';
             TR = obj.Prot.VFAData.Mat(:,2);
-            if ~isfield(data,'B1map'), data.B1map=1; end
-            [m0, t1] = mtv_compute_m0_t1(double(data.VFAData), flipAngles, TR(1), data.B1map);
-            FitResult.T1 = t1;
-            FitResult.M0 = m0;
-
-
+            if (length(unique(TR))~=1), error('VFA data must have same TR'); end
+            if ~isfield(data, 'B1map'), data.B1map = []; end
```

Comment:
I think I prefer if we keep the B1 map = 1 in the absence of a user-input B1 map, as it is essentially what the given assumption is (if no map is used, values are unchanged, so a B1 map with all values = 1 is essentially what is happening). The user can "view" this B1 map too on the GUI, so I find it helpful conceptually.

Same comment applies for the mask (if no mask is given, technically we are using a mask with all 1 values).

Reply:
i.e. the greater core qMRLab GUI software has use for those values, even though you hacked around them within your function by checking `isempty` and then setting B1 to 1 there (mask missing, see below). It should just be done here.

Reply:
Actuall nevermind, I see now that `data` isn't being returned to the object, so this is all fine.

Reply:
The `isempty()` part is from the old `mtv_compute_m0_t1()` function which I just copied over to the new function. 

My rationale was that since these functions already have code to handle missing roi and b1maps, it would be better to let them deal with it.